elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexTransaction.java

```diff
@@ -150,43 +139,6 @@ protected DataStoreTransaction getTransaction(Class<?> cls) {
     }
 
     @Override
-    public <T> Object getRelation(
```

Comment:
I'm with Aaron, I think these need to stay. (unless I'm missing something obvious this change totally borks the MultiplexTransaction)